tee-worker/cli/src/trusted_base_cli/commands/litentry/request_vc.rs

```diff
@@ -463,9 +462,9 @@ impl RequestVcCommand {
 		match perform_trusted_operation::<RequestVCResult>(cli, trusted_cli, &top) {
 			Ok(mut vc) => {
 				let decrypted = aes_decrypt(&key, &mut vc.vc_payload).unwrap();
-				let credential: Credential = serde_json::from_slice(&decrypted).unwrap();
+				let credential_str = String::from_utf8(decrypted).expect("Found invalid UTF-8");
```

Comment:
vc_payload is already an encoded json string, so I skipped the `serde_json` parsing